src/Tracy/BlueScreen/BlueScreen.php

```diff
@@ -360,17 +363,20 @@ public function isCollapsed(string $file): bool
 	public function getDumper(): \Closure
 	{
 		$keysToHide = array_flip(array_map('strtolower', $this->keysToHide));
+		$scrubber = $this->scrubber ?? function (string $k) use ($keysToHide): bool {
+			return isset($keysToHide[strtolower($k)]);
+		};
 
-		return function ($v, $k = null) use ($keysToHide): string {
-			if (is_string($k) && isset($keysToHide[strtolower($k)])) {
+		return function ($v, $k = null) use ($scrubber): string {
+			if (is_string($k) && call_user_func($scrubber, $k, $v)) {
```

Comment:
`$scrubber($k, $v)`

Reply:
Why? I'm not against, just don't know the reasoning.

Reply:
Okay, this seems to be a bad habit from PHP 5, even thogh it is only about negligible 1e-10 seconds faster, I updated the code.